### PR TITLE
data: pacifiquem.json added

### PR DESCRIPTION
**Pull request by pacifiquem**
## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5026"></a>

### CODE REVIEW

File: data/pacifiquem.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Pacifique Murangwa",
+  "avatar":"https://github.com/pacifiquem.png",
```
Comment (imbeshat) [inline data/pacifiquem.json:3]:
```suggestion
```

File: data/pacifiquem.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "type": "personal",
+  "bio": "Open Source enthusiast FullStack Developer",
+  "links": [
+    {
+      "name": "Github",
+      "url": "https://github.com/pacifiquem",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Linked",
```
Comment (imbeshat) [inline data/pacifiquem.json:13]:
```suggestion
      "name": "LinkedIn",
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
Please, update these.

**Comment by pacifiquem** [comment]
@imbeshat I've finished all suggested changes .

**Review by imbeshat — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**